sample/src/NimblePros.SampleToDo.Infrastructure/Data/Config/VogenEfCoreConverters.cs

```diff
@@ -8,8 +8,9 @@
 
 namespace NimblePros.SampleToDo.Infrastructure.Data.Config;
 
-[EfCoreConverter<ProjectName>] // not needed for structs
-[EfCoreConverter<ToDoItemId>] // not needed for structs
+[EfCoreConverter<ProjectName>] 
+[EfCoreConverter<ProjectId>]
```

Comment:
This was missing which was the main issue. I'm thinking of ways of making this easier: https://github.com/SteveDunn/Vogen/issues/681